x-pack/plugins/apm/public/components/shared/backend_link.tsx

```diff
@@ -28,6 +29,7 @@ export function BackendLink({
   subtype,
   type,
   onClick,
+  ...rest
```

Comment:
If this is not used, is this needed?

Reply:
hey @tan31989 have resolved this, please check. Thanks !